package.json

```diff
@@ -55,8 +53,10 @@
   },
   "dependencies": {
     "debug": "^4.1.1",
-    "deepmerge": "^3.2.0",
-    "ibm-watson": "^4.1.0"
+    "botbuilder-adapter-web": "^1.0.1",
```

Comment:
`botbuilder-adapter-web` is a dev dependency, because it is used in tests.

Reply:
fixed